src/main/kotlin/dev/tricht/gamesense/events/EventProducer.kt

```diff
@@ -13,7 +13,7 @@ import java.util.*
 class EventProducer : TimerTask() {
     private val dataFetcher = WindowsDataFetcher()
     private var client = ApiClientFactory().createApiClient()
-    private val dateFormat = DateFormat.getTimeInstance(0)
+    private val dateFormat = DateFormat.getTimeInstance()
```

Comment:
The `0` style argument is required. GE uses the VM option `-Djava.locale.providers=HOST` to support different locales (10:00:00 PM vs 22:00:00). If you add that to your IntelliJ run config, you'll see that it was working as expected.

Reply:
Gotcha - On my setup, with whatever versions of the various libraries were used, it caused the timezone info to show.  It works with your version.